src/main/java/com/orasi/bluesource/Employees.java

```diff
@@ -309,7 +312,100 @@ public boolean checkAccountPermissionOption(String strOption) {
 		catch (OptionNotInListboxException e){
 			return false;
 		}
-			
 	}
+	/**
+	 * Checks the manager dropdown for the employee name passed,
+	 * returns false when the name is not found and true when found
+	 * 
+	 * @param strOption the name of the employee you want to check for.
+	 * @return <code>true</code> if the name provided is available
+	 * in the dropdown, <code>false</code> otherwise.
+	 * @author Andrew McGrail
+	 */
+	
+	
+	public boolean checkManagerOption(String strOption) {
+		try{
+			lstManager.select(strOption);
+			return true;
+		}
+		catch (OptionNotInListboxException e){
+			return false;
+		}
+	}
+	
+	/**
+	 * This method returns the string of the success message
+	 * 
+	 * @return The success message in a String
+	 * @author Andrew McGrail
+	 */
+	public String getSuccessMessage() {
+		return driver.findElement(By.xpath("//*[@id='notification-area']/div")).getText().substring(2,35);
+	}
+	
+	/**
+	 * 
+	 * @return int - row number with the first non-Inactive User
+	 * @author Andrew McGrail
+	 */
+	public int findFirstActiveInTable(){
+		Integer intRow = null;
+		tblEmployees.syncVisible(2,true);
+		
+		//tblEmployees.getRowCount()
+		for (int i = 1; i < 50; i++) {
+			if (tblEmployees.getCellData(i, 12).equalsIgnoreCase("Permanent") | tblEmployees.getCellData(i, 12).equalsIgnoreCase("Contractor")) {
```

Comment:
To be honest, it gets confusing when choosing to use || or the | operator.  I think if you chose the single | operator, you are saying they both sides will be false in order for it to evaluate to false.  I think if you want either of them to be true for it to be true you need to use || ?  I might be wrong here, but just double check please. 
https://docs.oracle.com/javase/specs/jls/se7/html/jls-15.html#jls-15.22